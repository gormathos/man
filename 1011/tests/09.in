sol_id;user_id;task_id;score
134015;1029;2001;50.00
134018;1004;2003;0.00
134023;1027;2002;75.56
134028;1003;2002;15.56
134031;1004;2002;31.11
134032;1015;2001;61.54
134035;1009;2003;0.00
134035;1032;2003;6.25
134041;1011;2004;12.50
134042;1022;2005;32.61
134047;1006;2006;0.00
134051;1021;2004;33.33
134055;1016;2005;34.78
134064;1019;2006;21.21
134071;1030;2003;14.58
134073;1003;2006;0.00
134080;1017;2004;91.67
134083;1010;2003;29.17
134088;1019;2005;4.35
134095;1014;2005;0.00
134102;1032;2005;17.39
134112;1032;2004;45.83
134114;1021;2005;93.48
134118;1017;2001;19.23
134126;1010;2002;33.33
134131;1009;2005;21.74
134136;1004;2005;10.87
134141;1017;2001;19.23
134148;1006;2006;45.45
134153;1002;2001;11.54
134162;1006;2004;25.00
134163;1028;2002;28.89
134163;1017;2005;26.09
134170;1007;2002;64.44
134173;1012;2002;8.89
134181;1018;2004;79.17
134183;1021;2004;12.50
134186;1005;2006;87.88
134192;1020;2006;54.55
134192;1031;2005;56.52
134196;1012;2003;2.08
134205;1020;2003;31.25
134213;1017;2005;15.22
134215;1013;2002;2.22
134223;1015;2006;15.15
134231;1003;2005;2.17
134234;1015;2003;0.00
134239;1017;2004;50.00
134244;1024;2002;55.56
134251;1019;2001;88.46
134260;1027;2006;90.91
134269;1022;2005;10.87
134277;1006;2003;95.83
134287;1029;2005;43.48
134295;1012;2001;42.31
134302;1008;2006;54.55
134312;1020;2004;33.33
134313;1001;2006;51.52
134323;1014;2003;20.83
134331;1013;2004;12.50
134336;1020;2003;70.83
134338;1019;2001;46.15
134347;1025;2004;0.00
134352;1031;2001;7.69
134353;1016;2004;37.50
134356;1030;2006;9.09
134360;1021;2004;25.00
134360;1021;2003;0.00
134360;1021;2003;33.33
134362;1023;2004;25.00
134362;1006;2001;15.38
134363;1014;2001;73.08
134369;1018;2005;17.39
134376;1002;2003;58.33
134376;1016;2001;42.31
134383;1031;2006;72.73
134385;1031;2005;2.17
134391;1016;2003;43.75
134395;1002;2002;86.67
134401;1016;2004;0.00
134411;1004;2006;18.18
134414;1005;2005;52.17
134422;1022;2002;95.56
134432;1006;2002;0.00
134439;1005;2006;39.39
134448;1001;2003;50.00
134456;1006;2001;11.54
134457;1010;2004;4.17
134466;1008;2003;0.00
134468;1022;2001;3.85
134475;1009;2005;19.57
134483;1028;2006;60.61
134485;1007;2004;66.67
134491;1024;2001;15.38
134493;1019;2002;35.56
134493;1010;2002;93.33
134500;1029;2003;43.75
134510;1016;2004;16.67
134520;1003;2004;16.67
134522;1005;2006;9.09
134523;1001;2005;73.91
134528;1003;2005;41.30
134528;1030;2004;16.67
134534;1002;2004;75.00
134538;1018;2001;11.54
134543;1030;2006;54.55
134545;1003;2002;73.33
134549;1026;2002;73.33
134551;1001;2003;81.25
134555;1012;2002;4.44
134564;1026;2001;26.92
134565;1011;2005;0.00
134570;1007;2001;19.23
134579;1026;2005;45.65
134585;1023;2001;15.38
134592;1032;2005;69.57
134598;1024;2006;27.27
134608;1019;2002;2.22
134609;1030;2006;33.33
134610;1019;2005;17.39
134616;1024;2004;62.50
134619;1017;2002;15.56
134623;1010;2005;4.35
134625;1030;2002;35.56
134629;1013;2001;92.31
134630;1032;2001;46.15
134632;1029;2003;81.25
134639;1011;2006;51.52
134643;1002;2001;34.62
134651;1003;2006;33.33
134661;1013;2005;39.13
134664;1006;2003;16.67
134668;1019;2004;20.83
134668;1023;2005;23.91
134671;1017;2002;8.89
134673;1004;2005;10.87
134677;1029;2003;20.83
134687;1014;2005;26.09
134687;1031;2004;37.50
134691;1025;2006;6.06
134696;1013;2001;19.23
134698;1002;2004;0.00
134702;1025;2001;92.31
134712;1020;2006;36.36
134720;1025;2006;24.24
134727;1020;2006;15.15
134727;1006;2006;100.00
134730;1006;2006;45.45
134733;1005;2004;0.00
134736;1001;2003;31.25
134736;1030;2005;69.57
134744;1007;2002;51.11
134752;1004;2003;22.92
134754;1009;2001;0.00
134761;1009;2003;22.92
134768;1004;2003;0.00
134769;1029;2006;9.09
134772;1010;2003;52.08
134782;1007;2004;37.50
134792;1021;2001;53.85
134797;1025;2002;77.78
134801;1027;2004;8.33
134808;1015;2006;24.24
134812;1028;2004;54.17
134814;1020;2001;23.08
134814;1021;2001;73.08
134821;1002;2005;97.83
134821;1018;2003;6.25
134822;1003;2003;41.67
134824;1019;2003;64.58
134827;1022;2003;43.75
134837;1020;2004;12.50
134837;1008;2005;97.83
134840;1002;2003;20.83
134843;1006;2001;0.00
134853;1032;2006;33.33
134863;1014;2006;15.15
134870;1001;2002;22.22
134873;1032;2002;8.89
134875;1007;2005;84.78
134883;1015;2004;0.00
134891;1001;2006;15.15
134899;1018;2004;50.00
134908;1011;2002;33.33
134912;1008;2003;8.33
134916;1007;2002;20.00
134918;1008;2004;95.83
134921;1012;2003;72.92
134924;1027;2003;70.83
134933;1014;2003;52.08
134934;1024;2002;17.78
134944;1018;2003;56.25
134954;1020;2003;91.67
134961;1022;2006;6.06
134965;1017;2001;0.00
134967;1006;2001;0.00
134975;1011;2002;44.44
134980;1002;2002;15.56
134986;1030;2001;84.62
134987;1017;2001;69.23
134994;1003;2004;37.50
134995;1015;2001;0.00
135005;1029;2002;31.11
135014;1021;2003;4.17
135019;1021;2006;0.00
135021;1021;2001;11.54
135022;1002;2006;9.09
135030;1027;2001;69.23
135040;1001;2001;15.38
135042;1020;2002;6.67
135047;1004;2001;11.54
135052;1021;2002;33.33
135060;1023;2001;69.23
135064;1007;2003;64.58
135072;1031;2002;28.89
135075;1025;2006;0.00
135075;1023;2003;43.75
135085;1025;2005;43.48
135093;1016;2002;11.11
135095;1032;2003;54.17
135098;1005;2004;45.83
135100;1003;2006;72.73
135110;1025;2003;14.58
135119;1011;2005;78.26
135127;1021;2005;41.30
135128;1029;2002;24.44
135129;1013;2001;50.00
135135;1018;2002;8.89
135143;1010;2006;57.58
135145;1004;2004;75.00
135153;1028;2002;20.00
135154;1029;2004;12.50
135161;1019;2006;75.76
135162;1028;2003;81.25
135167;1022;2002;4.44
135167;1013;2001;88.46
135174;1016;2001;23.08
135174;1008;2003;35.42
135175;1024;2002;17.78
135180;1013;2001;26.92
135189;1024;2004;41.67
135197;1027;2006;30.30
135204;1029;2002;80.00
135207;1025;2003;0.00
135211;1032;2006;66.67
135217;1014;2006;21.21
135223;1031;2002;57.78
135228;1011;2003;27.08
135228;1014;2002;100.00
135237;1022;2006;72.73
135247;1022;2006;21.21
135255;1032;2003;4.17
135260;1024;2003;68.75
135261;1016;2005;6.52
135262;1032;2003;27.08
135266;1014;2005;19.57
135274;1003;2001;42.31
135281;1006;2004;0.00
135284;1024;2005;30.43
135287;1030;2005;60.87
135294;1016;2006;57.58
135300;1004;2003;0.00
135305;1029;2003;35.42
135307;1016;2004;0.00
135312;1014;2001;30.77
135313;1015;2006;9.09
135323;1011;2001;76.92
135330;1002;2003;16.67
135339;1015;2002;4.44
135344;1023;2006;3.03
135354;1003;2005;73.91
135359;1020;2002;2.22
135366;1023;2006;36.36
135366;1027;2005;8.70
135366;1004;2004;70.83
135371;1031;2001;11.54
135375;1006;2006;0.00
135377;1030;2001;7.69
135386;1018;2003;93.75
135395;1025;2005;76.09
135403;1025;2003;45.83
135412;1005;2005;39.13
135414;1025;2002;24.44
135417;1019;2006;21.21
135423;1027;2005;4.35
135431;1028;2001;3.85
135436;1008;2001;11.54
135440;1026;2001;0.00
135444;1031;2006;6.06
135447;1016;2006;84.85
135455;1018;2006;9.09
135460;1002;2005;34.78
135464;1009;2003;12.50
135464;1005;2003;83.33
135473;1004;2006;24.24
135479;1027;2001;7.69
135489;1017;2003;16.67
135499;1023;2006;54.55
135504;1011;2002;91.11
135509;1008;2003;8.33
135510;1009;2006;72.73
135513;1020;2002;44.44
135514;1032;2002;20.00
135516;1013;2005;0.00
135523;1011;2004;95.83
135528;1011;2005;17.39
135535;1009;2004;50.00
135545;1013;2005;13.04
135550;1003;2002;8.89
135556;1016;2002;20.00
135566;1006;2002;37.78
135566;1009;2003;20.83
135571;1025;2004;12.50
135571;1006;2002;22.22
135574;1004;2004;45.83
135577;1025;2003;72.92
135584;1022;2005;60.87
135592;1028;2004;25.00
135592;1007;2001;7.69
135602;1004;2005;56.52
135608;1014;2003;29.17
135608;1030;2003;29.17
135618;1026;2006;60.61
135621;1026;2001;0.00
135621;1007;2002;35.56
135624;1010;2004;0.00
135631;1003;2001;0.00
135635;1011;2004;4.17
135644;1004;2006;0.00
135651;1015;2005;28.26
135658;1017;2002;24.44
135664;1027;2003;2.08
135670;1017;2004;0.00
135670;1013;2006;12.12
135672;1021;2004;79.17
135672;1018;2004;25.00
135674;1005;2001;34.62
135682;1028;2001;11.54
135684;1007;2002;24.44
135694;1001;2002;51.11
135699;1014;2002;4.44
135700;1007;2004;100.00
135702;1027;2004;12.50
135703;1007;2006;36.36
135713;1008;2004;12.50
135723;1004;2005;45.65
135731;1028;2004;25.00
135736;1021;2005;58.70
135740;1010;2005;52.17
135742;1017;2003;20.83
135751;1024;2006;12.12
135759;1030;2001;30.77
135759;1011;2006;6.06
135759;1020;2005;0.00
135759;1008;2004;25.00
135762;1019;2003;37.50
135762;1005;2001;15.38
135764;1015;2004;33.33
135772;1023;2003;62.50
135777;1003;2006;3.03
135786;1012;2002;0.00
135792;1003;2004;0.00
135794;1006;2004;58.33
135804;1006;2005;0.00
135807;1025;2004;29.17
135807;1008;2006;33.33
135813;1018;2002;24.44
135814;1024;2005;43.48
135819;1026;2002;57.78
135829;1002;2005;43.48
135839;1007;2001;3.85
135848;1009;2006;93.94
135854;1005;2003;8.33
135860;1001;2001;3.85
135867;1011;2005;19.57
135870;1017;2004;8.33
135875;1030;2006;24.24
135879;1020;2005;4.35
135881;1009;2005;4.35
135886;1029;2001;7.69
135892;1003;2004;100.00
135897;1012;2005;23.91
135900;1010;2001;69.23
135908;1004;2002;53.33
135917;1014;2004;25.00
135923;1030;2002;2.22
135929;1024;2004;4.17
135929;1015;2003;83.33
135932;1023;2004;70.83
135933;1014;2001;42.31
135940;1015;2003;43.75
135940;1031;2002;0.00
135940;1002;2004;8.33
135944;1005;2003;31.25
135950;1022;2002;6.67
135958;1017;2005;13.04
135958;1010;2003;27.08
135966;1016;2003;39.58
135970;1002;2002;80.00
135980;1001;2004;54.17
135982;1015;2004;66.67
135989;1029;2004;37.50
135991;1028;2006;18.18
135995;1014;2006;27.27
136005;1021;2002;20.00
136011;1028;2003;27.08
136012;1029;2005;2.17
136015;1025;2005;34.78
136023;1016;2003;2.08
136031;1030;2004;4.17
136034;1001;2006;3.03
136041;1012;2005;45.65
136046;1016;2003;6.25
136048;1001;2005;28.26
136050;1028;2001;57.69
136058;1027;2005;2.17
136068;1025;2001;50.00
136075;1029;2001;84.62
136075;1003;2002;22.22
136083;1017;2003;72.92
136090;1029;2004;29.17
136099;1013;2003;62.50
136103;1015;2003;50.00
136105;1008;2002;33.33
136115;1024;2005;6.52
136119;1026;2001;61.54
136126;1002;2004;4.17
136131;1028;2001;38.46
136141;1003;2006;15.15
136151;1010;2006;3.03
136160;1003;2004;16.67
136169;1006;2005;39.13
136174;1010;2001;53.85
136174;1007;2004;16.67
136178;1026;2002;33.33
136186;1027;2006;39.39
136194;1008;2003;22.92
136194;1017;2003;6.25
136204;1016;2006;3.03
136211;1026;2004;54.17
136215;1031;2006;0.00
136217;1024;2003;33.33
136223;1025;2001;19.23
136227;1004;2001;84.62
136237;1019;2006;36.36
136239;1015;2006;33.33
136241;1030;2003;43.75
136248;1019;2003;8.33
136256;1022;2003;54.17
136257;1031;2003;62.50
136265;1007;2005;43.48
136269;1031;2002;66.67
136269;1015;2002;44.44
136271;1002;2005;6.52
136278;1015;2005;13.04
136283;1003;2005;21.74
136283;1007;2003;10.42
136291;1032;2004;4.17
136298;1016;2003;68.75
136308;1029;2002;40.00
136315;1026;2004;4.17
136325;1007;2001;50.00
136331;1013;2003;12.50
136341;1027;2005;84.78
136351;1027;2003;77.08
136357;1027;2002;91.11
136366;1022;2003;27.08
136374;1002;2001;7.69
136380;1001;2002;20.00
136386;1011;2003;14.58
136387;1006;2003;64.58
136395;1007;2003;100.00
136396;1021;2002;64.44
136404;1010;2002;6.67
136413;1026;2004;4.17
136419;1026;2003;56.25
136429;1003;2001;73.08
136436;1018;2005;76.09
136441;1024;2003;14.58
136451;1006;2001;0.00
136459;1009;2004;0.00
136462;1010;2002;77.78
136472;1024;2006;3.03
136476;1031;2002;51.11
136486;1018;2004;8.33
136486;1011;2006;75.76
136491;1024;2001;0.00
136497;1010;2002;22.22
136501;1010;2005;0.00
136509;1021;2006;15.15
136512;1026;2003;6.25
136517;1006;2003;31.25
136519;1012;2001;3.85
136522;1026;2005;0.00
136524;1026;2002;60.00
136524;1005;2003;18.75
136533;1016;2002;48.89
136534;1009;2001;84.62
136535;1014;2005;80.43
136541;1009;2006;30.30
136544;1030;2004;12.50
136553;1009;2002;0.00
136557;1012;2003;18.75
136567;1019;2001;3.85
136572;1014;2002;84.44
136572;1002;2004;8.33
136573;1016;2006;72.73
136583;1012;2002;51.11
136586;1023;2003;6.25
136588;1021;2001;30.77
136592;1005;2004;16.67
136598;1011;2003;85.42
136606;1022;2004;16.67
136609;1003;2002;24.44
136615;1016;2001;61.54
136619;1032;2005;52.17
136627;1008;2001;46.15
136628;1032;2004;62.50
136635;1007;2001;46.15
136641;1030;2005;6.52
136645;1023;2005;47.83
136649;1027;2004;16.67
136657;1018;2001;96.15
136662;1012;2004;16.67
136671;1027;2006;93.94
136679;1009;2004;87.50
136687;1028;2003;10.42
136695;1023;2001;19.23
136698;1031;2003;29.17
136699;1021;2006;63.64
136701;1006;2003;41.67
136710;1027;2001;50.00
136713;1020;2001;38.46
136721;1015;2001;3.85
136731;1032;2002;31.11
136733;1023;2005;36.96
136733;1006;2005;76.09
136739;1019;2004;62.50
136742;1031;2001;19.23
136747;1003;2001;7.69
136754;1012;2002;17.78
136758;1031;2005;26.09
136762;1002;2001;34.62
136771;1021;2002;2.22
136781;1005;2006;75.76
136790;1010;2004;41.67
136794;1029;2006;78.79
136799;1018;2002;4.44
136809;1008;2001;80.77
136814;1008;2001;0.00
136820;1021;2005;0.00
136830;1032;2006;63.64
136831;1026;2005;15.22
136837;1023;2002;71.11
136838;1010;2006;21.21
136842;1012;2004;25.00
136852;1015;2002;11.11
136857;1022;2004;33.33
136867;1006;2005;4.35
136873;1017;2003;29.17
136873;1030;2001;23.08
136877;1008;2006;0.00
136883;1012;2005;65.22
136884;1028;2002;97.78
136890;1027;2004;70.83
136900;1019;2004;95.83
136904;1004;2003;12.50
136907;1020;2001;42.31
136912;1005;2002;88.89
136914;1020;2006;33.33
136918;1001;2001;3.85
136921;1023;2004;16.67
136924;1031;2005;58.70
136926;1013;2006;9.09
136926;1032;2004;37.50
136934;1017;2002;86.67
136936;1011;2002;68.89
136940;1028;2002;33.33
136941;1028;2004;16.67
136943;1031;2003;0.00
136948;1021;2005;21.74
136949;1022;2006;12.12
136954;1024;2001;11.54
136956;1009;2001;76.92
136960;1018;2006;0.00
136968;1025;2004;87.50
136976;1020;2006;36.36
136982;1025;2002;48.89
136991;1009;2003;2.08
137001;1026;2006;30.30
137009;1002;2005;39.13
137016;1006;2004;8.33
137026;1014;2003;91.67
137032;1022;2001;88.46
137032;1006;2002;0.00
137037;1026;2004;20.83
137044;1020;2002;95.56
137054;1030;2002;31.11
137056;1031;2004;45.83
137059;1015;2001;0.00
137063;1028;2005;36.96
137068;1010;2003;62.50
137073;1019;2002;55.56
137078;1020;2005;6.52
137078;1004;2002;66.67
137085;1031;2001;80.77
137093;1008;2002;8.89
137102;1028;2005;19.57
137104;1013;2002;33.33
137113;1023;2006;12.12
137116;1030;2002;62.22
137123;1005;2005;82.61
137127;1013;2005;97.83
137136;1014;2002;28.89
137138;1020;2001;34.62
137145;1004;2004;41.67
137154;1024;2004;12.50
137163;1008;2006;9.09
137168;1008;2005;10.87
137172;1028;2002;40.00
137174;1008;2005;4.35
137174;1014;2005;0.00
137181;1006;2002;40.00
137189;1026;2002;4.44
137195;1029;2006;0.00
137197;1005;2002;91.11
137200;1027;2005;6.52
137207;1001;2001;19.23
137208;1013;2004;12.50
137208;1023;2002;80.00
137212;1029;2001;3.85
137219;1022;2001;15.38
137226;1022;2001;30.77
137233;1018;2005;10.87
137233;1015;2003;4.17
137239;1018;2001;53.85
137246;1016;2005;36.96
137254;1022;2002;20.00
137262;1002;2006;3.03
137268;1019;2002;20.00
137278;1022;2005;32.61
137278;1001;2005;56.52
137288;1026;2005;52.17
137293;1008;2002;2.22
137298;1015;2005;2.17
137305;1029;2002;0.00
137314;1004;2005;69.57
137324;1012;2006;15.15
137326;1031;2003;0.00
137328;1014;2001;0.00
137332;1002;2003;81.25
137341;1001;2004;45.83
137350;1002;2006;39.39
137352;1032;2004;0.00
137356;1020;2003;8.33
137360;1024;2006;24.24
137361;1028;2004;4.17
137365;1023;2002;60.00
137369;1005;2002;2.22
137378;1032;2003;33.33
137378;1020;2004;58.33
137382;1009;2002;53.33
137382;1021;2003;0.00
137392;1025;2003;6.25
137400;1010;2001;84.62
137409;1017;2002;8.89
137413;1003;2003;0.00
137423;1009;2006;21.21
137432;1016;2005;58.70
137438;1016;2001;7.69
137445;1019;2005;100.00
137445;1001;2004;100.00
137454;1022;2002;68.89
137462;1008;2006;0.00
137463;1032;2006;36.36
137466;1027;2001;3.85
137476;1011;2006;6.06
137486;1009;2001;88.46
137487;1012;2003;41.67
137496;1020;2002;0.00
137501;1009;2004;41.67
137508;1002;2001;92.31
137512;1012;2005;2.17
137515;1014;2004;41.67
137518;1003;2003;8.33
137527;1001;2006;3.03
137533;1020;2004;12.50
137537;1025;2006;36.36
137544;1024;2001;23.08
137544;1029;2004;79.17
137544;1007;2003;91.67
137554;1005;2003;56.25
137559;1016;2005;8.70
137562;1009;2002;8.89
137567;1025;2004;8.33
137570;1026;2003;16.67
137573;1031;2004;8.33
137573;1007;2003;33.33
137577;1010;2006;69.70
137583;1015;2001;38.46
137583;1002;2006;60.61
137585;1021;2001;84.62
137591;1004;2004;91.67
137601;1026;2006;72.73
137609;1031;2006;42.42
137616;1015;2004;70.83
137624;1013;2002;80.00
137629;1009;2004;29.17
137636;1013;2006;81.82
137644;1005;2006;15.15
137649;1025;2002;20.00
137656;1022;2003;20.83
137662;1015;2006;39.39
137671;1030;2005;2.17
137676;1029;2006;30.30
137679;1005;2001;26.92
137686;1013;2005;30.43
137696;1024;2006;27.27
137697;1017;2006;6.06
137701;1013;2006;27.27
137703;1001;2001;0.00
137707;1032;2005;84.78
137710;1015;2004;91.67
137719;1004;2002;33.33
137725;1012;2003;0.00
137733;1007;2005;2.17
137743;1005;2004;45.83
137751;1025;2005;52.17
137753;1010;2004;25.00
137761;1025;2001;15.38
137771;1014;2004;0.00
137772;1006;2004;33.33
137781;1020;2005;2.17
137789;1005;2001;42.31
137789;1011;2003;10.42
137796;1019;2005;58.70
137796;1014;2006;45.45
137803;1004;2006;39.39
137810;1010;2005;73.91
137815;1003;2001;11.54
137823;1027;2001;7.69
137825;1001;2005;0.00
137827;1024;2002;6.67
137830;1023;2006;18.18
137832;1017;2006;12.12
137839;1021;2006;33.33
137842;1031;2004;16.67
137846;1024;2003;18.75
137854;1007;2006;54.55
137862;1018;2005;0.00
137869;1010;2001;11.54
137877;1004;2001;26.92
137878;1005;2001;61.54
137884;1026;2001;76.92
137884;1025;2001;3.85
137889;1008;2004;20.83
137890;1003;2003;6.25
137896;1030;2003;68.75
137905;1018;2005;6.52
137914;1008;2005;45.65
137920;1020;2003;66.67
137921;1030;2002;57.78
137925;1002;2006;21.21
137928;1015;2005;19.57
137930;1014;2004;45.83
137930;1007;2005;2.17
137938;1024;2002;28.89
137943;1029;2005;4.35
137943;1017;2005;15.22
137950;1014;2002;17.78
137959;1029;2006;6.06
137964;1023;2003;12.50
137973;1025;2006;78.79
137977;1020;2004;83.33
137982;1008;2005;0.00
137987;1018;2002;73.33
137996;1020;2005;0.00
137999;1024;2004;0.00
138000;1029;2004;83.33
138010;1024;2005;15.22
138013;1012;2006;81.82
138023;1017;2004;8.33
138024;1021;2002;37.78
138028;1027;2004;8.33
138038;1003;2005;54.35
138040;1027;2002;6.67
138044;1012;2006;15.15
138054;1008;2004;0.00
138061;1029;2005;8.70
138069;1012;2001;53.85
138073;1004;2002;24.44
138078;1005;2005;10.87
138081;1032;2006;6.06
138088;1030;2005;50.00
138093;1027;2006;3.03
138094;1001;2004;45.83
138101;1019;2003;0.00
138101;1004;2001;23.08
138107;1022;2004;91.67
138107;1011;2002;4.44
138111;1029;2001;96.15
138114;1013;2006;18.18
138118;1024;2003;2.08
138126;1008;2002;100.00
138134;1018;2006;9.09
138138;1001;2002;51.11
138145;1012;2004;75.00
138155;1012;2005;28.26
138164;1009;2002;73.33
138164;1030;2004;75.00
138172;1021;2003;52.08
138176;1011;2001;46.15
138181;1001;2002;93.33
138188;1019;2002;13.33
138198;1023;2001;100.00
138203;1007;2006;12.12
138209;1004;2006;60.61
138211;1013;2003;47.92
138213;1010;2004;50.00
138216;1030;2006;30.30
138224;1012;2004;0.00
138228;1023;2005;47.83
138231;1016;2001;19.23
138235;1019;2003;10.42
138239;1030;2001;23.08
138247;1014;2006;69.70
138251;1013;2004;25.00
138252;1028;2005;17.39
138256;1028;2001;7.69
138260;1003;2003;83.33
138262;1024;2001;50.00
138262;1026;2004;0.00
138264;1012;2006;66.67
138265;1006;2005;2.17
138275;1023;2001;0.00
138277;1032;2001;57.69
138285;1032;2002;24.44
138292;1018;2001;84.62
138298;1019;2004;12.50
138301;1005;2005;23.91
138301;1017;2006;18.18
138308;1005;2002;82.22
138311;1023;2004;54.17
138314;1022;2005;10.87
138323;1018;2002;0.00
138328;1007;2005;95.65
138336;1001;2003;39.58
138346;1015;2002;82.22
138350;1027;2003;25.00
138360;1026;2006;100.00
138360;1010;2003;52.08
138366;1017;2006;0.00
138374;1031;2003;2.08
138378;1008;2001;11.54
138381;1027;2003;35.42
138390;1002;2003;8.33
138396;1024;2005;50.00
138403;1016;2002;0.00
138410;1030;2003;43.75
138418;1011;2001;69.23
138425;1011;2006;0.00
138432;1013;2002;84.44
138432;1011;2001;53.85
138439;1014;2003;2.08
138446;1021;2006;36.36
138447;1007;2006;63.64
138452;1032;2001;7.69
138458;1019;2006;51.52
138467;1009;2005;76.09
138472;1032;2001;0.00
138478;1028;2005;2.17
138486;1012;2004;79.17
138488;1005;2002;86.67
138495;1011;2004;25.00
138495;1023;2002;22.22
138501;1022;2004;87.50
138511;1007;2006;39.39
138511;1010;2006;90.91
138512;1017;2005;58.70
138516;1028;2006;54.55
138526;1004;2001;11.54
138530;1028;2006;51.52
138536;1025;2002;55.56
138541;1032;2001;34.62
138542;1009;2001;11.54
138543;1023;2004;4.17
138552;1023;2005;15.22
138562;1027;2002;64.44
138564;1030;2004;4.17
138569;1012;2006;3.03
138570;1017;2001;30.77
138577;1009;2005;6.52
138581;1012;2001;80.77
138586;1002;2002;26.67
138591;1023;2002;62.22
138591;1009;2002;4.44
138591;1013;2003;0.00
138593;1011;2005;65.22
138594;1029;2003;22.92
138602;1001;2004;58.33
138608;1001;2005;69.57
138617;1014;2001;42.31
138618;1019;2004;66.67
138626;1028;2003;14.58
138634;1022;2004;79.17
138638;1028;2006;93.94
138648;1011;2003;81.25
138650;1001;2006;12.12
138650;1002;2002;24.44
138659;1007;2004;29.17
138659;1018;2006;3.03
138666;1027;2002;2.22
138669;1019;2001;76.92
138678;1013;2004;0.00
138683;1019;2001;26.92
138691;1022;2006;33.33
138692;1032;2002;28.89
138695;1013;2004;95.83
138700;1031;2006;6.06
138708;1021;2004;100.00
138716;1018;2006;45.45
138721;1026;2006;100.00
138729;1013;2002;2.22
138731;1005;2004;0.00
138735;1009;2006;24.24
138736;1028;2003;62.50
138744;1018;2001;80.77
138749;1010;2005;39.13
138759;1026;2003;68.75
138764;1001;2003;35.42
138772;1008;2002;28.89
138781;1015;2002;31.11
138783;1010;2001;0.00
138786;1032;2005;71.74
138791;1022;2001;3.85
138792;1029;2005;15.22
138802;1014;2004;20.83
138812;1016;2002;28.89
138813;1016;2006;36.36
138813;1022;2003;93.75
138818;1031;2001;7.69
138823;1023;2003;62.50
138823;1011;2004;0.00
138830;1018;2003;14.58
138833;1017;2006;81.82
138836;1026;2005;50.00
138845;1031;2005;30.43
138848;1013;2003;87.50
138849;1031;2004;8.33
138856;1016;2004;16.67
138866;1018;2004;4.17
138866;1028;2005;71.74
138867;1026;2003;41.67
138868;1012;2001;23.08
138870;1018;2003;52.08
138878;1025;2005;0.00
138880;1015;2005;8.70
138888;1019;2005;13.04
138891;1020;2001;26.92
138891;1011;2001;34.62
