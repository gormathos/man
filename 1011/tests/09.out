1001: Very Good [69.87%]
1002: Excellent [82.28%]
1003: Excellent [79.40%]
1004: Very Good [66.01%]
1005: Excellent [75.38%]
1006: Very Good [64.27%]
1007: Excellent [78.96%]
1008: Excellent [77.40%]
1009: Very Good [73.71%]
1010: Excellent [75.88%]
1011: Excellent [83.88%]
1012: Very Good [71.84%]
1013: Excellent [89.96%]
1014: Excellent [76.78%]
1015: Very Good [64.40%]
1016: Very Good [60.04%]
1017: Excellent [76.83%]
1018: Excellent [77.32%]
1019: Excellent [80.03%]
1020: Very Good [62.32%]
1021: Excellent [76.38%]
1022: Excellent [83.84%]
1023: Very Good [69.28%]
1024: Participant [52.35%]
1025: Excellent [80.90%]
1026: Very Good [70.89%]
1027: Excellent [81.16%]
1028: Excellent [76.09%]
1029: Excellent [77.17%]
1030: Very Good [69.12%]
1031: Very Good [64.53%]
1032: Participant [59.49%]
